<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectBrowserPage</class>
 <widget class="QWidget" name="ProjectBrowserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>497</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure project viewer settings&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line8">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colours</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Highlighted entries (Others):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbHighlightedButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the colour for highlighted entries in the Others viewer.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pbGroup">
     <property name="title">
      <string>Visible Project Browsers</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Projecttype:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projectTypeCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select the project type to be configured</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="sourcesBrowserCheckBox">
          <property name="toolTip">
           <string>Select to show the sources browser</string>
          </property>
          <property name="text">
           <string>Sources Browser</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="translationsBrowserCheckBox">
          <property name="toolTip">
           <string>Select to show the translations browser</string>
          </property>
          <property name="text">
           <string>Translations Browser</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="formsBrowserCheckBox">
          <property name="toolTip">
           <string>Select to show the forms browser</string>
          </property>
          <property name="text">
           <string>Forms Browser</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="interfacesBrowserCheckBox">
          <property name="toolTip">
           <string>Select to show the interfaces (IDL) browser</string>
          </property>
          <property name="text">
           <string>Interfaces (IDL) Browser</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="resourcesBrowserCheckBox">
          <property name="toolTip">
           <string>Select to show the resources browser</string>
          </property>
          <property name="text">
           <string>Resources Browser</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="othersBrowserCheckBox">
          <property name="toolTip">
           <string>Select to show the browser for other files</string>
          </property>
          <property name="text">
           <string>Others Browser</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="followEditorCheckBox">
     <property name="toolTip">
      <string>Select to make the project browsers highlight the file of the current editor.</string>
     </property>
     <property name="text">
      <string>Highlight file of current editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="followCursorLineCheckBox">
     <property name="toolTip">
      <string>Select to follow the cursor line of the current editor</string>
     </property>
     <property name="text">
      <string>Follow cursor line of current editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoPopulateCheckBox">
     <property name="toolTip">
      <string>Select to populate entries automatically when following the cursor line of the current editor</string>
     </property>
     <property name="text">
      <string>Populate entry when following the cursor line</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hideGeneratedCheckBox">
     <property name="toolTip">
      <string>Select to hide sources generated from form files</string>
     </property>
     <property name="text">
      <string>Hide generated form sources</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>599</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbHighlightedButton</tabstop>
  <tabstop>projectTypeCombo</tabstop>
  <tabstop>sourcesBrowserCheckBox</tabstop>
  <tabstop>formsBrowserCheckBox</tabstop>
  <tabstop>resourcesBrowserCheckBox</tabstop>
  <tabstop>translationsBrowserCheckBox</tabstop>
  <tabstop>interfacesBrowserCheckBox</tabstop>
  <tabstop>othersBrowserCheckBox</tabstop>
  <tabstop>followEditorCheckBox</tabstop>
  <tabstop>followCursorLineCheckBox</tabstop>
  <tabstop>autoPopulateCheckBox</tabstop>
  <tabstop>hideGeneratedCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
